product_docs/docs/pgd/5/manual/06-check-cluster.mdx

```diff
@@ -0,0 +1,110 @@
+---
+title: Step 6 - Checking the cluster
+navTitle: Checking the cluster
+deepToC: true
+---
+
+## Checking the cluster
+
+With the cluster up and running, it is worth running some basic checks on how effectively it is replicating.
+
+The simplest way to test the cluster is replicating is to log into one node, create a table and populate it.
+Then log into another node and confirm that the table schema and content is replicated on that node.
+
+In the following worked example, we show one quick way to do this but you should ensure that any testing you perform is appropriate for your use case.
+
+## Worked example
+
+### Preparation
+
+Log into host-one's Postgres server.
+```
+ssh admin@host-one
+sudo -iu enterprisedb psql bdrdbdoc
+```
+
+This is your connection to PGD's node-one.
+
+To ensure that the cluster is ready to go, run: 
+
+```
+select bdr.wait_slot_confirm_lsn(NULL, NULL)
+```
```

Comment:
Here, I think providing an example output would be helpful. What result do I get that confirms that the cluster is ready?